lib/graphql/fragment_cache/railtie.rb

```diff
@@ -10,6 +10,9 @@ class Railtie < ::Rails::Railtie # :nodoc:
       module Config
         class << self
           def store=(store)
+            cache_format_version = Rails.application&.config&.active_support&.cache_format_version
```

Comment:
should we do that only when rails version is bigger than 7.0?

Reply:
@DmitryTsepelev 
Yes, that's correct, thanks!
I modified to update `ActiveSupport::Cache.format_version` when rails version is 7.0 or higher.
https://github.com/DmitryTsepelev/graphql-ruby-fragment_cache/pull/115/commits/5136021290548932804732ba6b2362fa75cc6283